### PR TITLE
chore: bump sqlx to 0.7.1

### PR DESCRIPTION
**Pull request by Arqu**
As per https://github.com/jbr/async-sqlx-session/pull/32

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,13 +24,14 @@ async_std = ["async-std"]
 mysql = ["sqlx/mysql", "sqlx/json"]
 
 [dependencies]
-async-session = "3.0.0"
-sqlx = { version = "0.6.2", features = ["chrono"] }
+async-session = { git = "https://github.com/http-rs/async-session", branch = "main"}
```
Comment (maxcountryman) [inline Cargo.toml:27]: This seems fairly dangerous.

File: Cargo.toml [hunk 2]
```diff
@@ -24,13 +24,14 @@ async_std = ["async-std"]
...
+sqlx = { version = "0.7.1", features = [ "time" ] }
```
Comment (maxcountryman) [inline Cargo.toml:28]: What's the logic for switching to `time` here? That seems unrelated to upgrading.

File: Cargo.toml (hunk 1 above)
Comment (Arqu) [reply Cargo.toml:27]: I agree, on the other hand, there's been no release in over 2 years and some of the deps updates are required for `sqlx 0.7.1`

File: Cargo.toml (hunk 2 above)
Comment (Arqu) [reply Cargo.toml:28]: Due to `async-session` moving away from `chrono` and to `time`
https://github.com/http-rs/async-session/pull/35

**Review by Arqu — COMMENTED** [review]
Hope this makes sense. Ideally I'd move `async-session` to a release. Filed https://github.com/http-rs/async-session/issues/63 to nudge things along. Other than that I can just pin it to a commit which is slightly less risky.
